doesn't work locally for me (neither does `main`).

```
RUN_SLOW=1 pytest tests/models/vitmatte/
=================================================================================================================================== test session starts ===================================================================================================================================
platform darwin -- Python 3.9.19, pytest-8.2.0, pluggy-1.5.0
rootdir: /Users/richard.brown/Downloads/transformers
configfile: pyproject.toml
collected 121 items / 1 error                                                                                                                                                                                                                                                             

========================================================================================================================================= ERRORS ==========================================================================================================================================
________________________________________________________________________________________________________ ERROR collecting tests/models/vitmatte/test_image_processing_vitmatte.py _________________________________________________________________________________________________________
../../Documents/Code/miniconda3/envs/temp/lib/python3.9/unittest/case.py:462: in __hash__
    return hash((type(self), self._testMethodName))
E   AttributeError: 'VitMatteImageProcessingTester' object has no attribute '_testMethodName'
==================================================================================================================================== warnings summary =====================================================================================================================================
../../Documents/Code/miniconda3/envs/temp/lib/python3.9/site-packages/_pytest/config/__init__.py:1448
  /Users/richard.brown/Documents/Code/miniconda3/envs/temp/lib/python3.9/site-packages/_pytest/config/__init__.py:1448: PytestConfigWarning: Unknown config option: doctest_glob
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================================================================================================================================= short test summary info =================================================================================================================================
ERROR tests/models/vitmatte/test_image_processing_vitmatte.py::VitMatteImageProcessingTester - AttributeError: 'VitMatteImageProcessingTester' object has no attribute '_testMethodName'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=============================================================================================================================== 1 warning, 1 error in 8.19s ===============================================================================================================================

```

--- a/src/transformers/models/vitmatte/image_processing_vitmatte.py
+++ b/src/transformers/models/vitmatte/image_processing_vitmatte.py
@@ -133,9 +133,9 @@ class VitMatteImageProcessor(BaseImageProcessor):
 
         height, width = get_image_size(image, input_data_format)
 
-        if height % size_divisibility != 0 or width % size_divisibility != 0:
-            pad_height = size_divisibility - height % size_divisibility
-            pad_width = size_divisibility - width % size_divisibility
+        pad_height = 0 if height % size_divisibility == 0 else size_divisibility - height % size_divisibility
+        pad_width = 0 if width % size_divisibility == 0 else size_divisibility - width % size_divisibility
+        if pad_width + pad_height > 0:
             padding = ((0, pad_height), (0, pad_width))
             image = pad(image, padding=padding, data_format=data_format, input_data_format=input_data_format)
 

--- a/tests/models/vitmatte/test_image_processing_vitmatte.py
+++ b/tests/models/vitmatte/test_image_processing_vitmatte.py
@@ -192,3 +192,7 @@ class VitMatteImageProcessingTest(ImageProcessingTestMixin, unittest.TestCase):
         image = np.random.randn(3, 249, 491)
         images = image_processing.pad_image(image)
         assert images.shape == (3, 256, 512)
+
+        image = np.random.randn(3, 249, 512)
+        images = image_processing.pad_image(image)
+        assert images.shape == (3, 256, 512)

